### DISCUSSION TITLE
Des anyone know if we can load Big Science - Bloom into this

### DISCUSSION THREAD

**Discussion by TheMindExpansionNetwork** [oobabooga/text-generation-webui#525] [Q&A]
I found this cool sounding model it's pretty big in safe tensor and wanted to know if that would be supported if I install the entire stack. 

I am working on bunch of cool stuff like keeping AI internal. 

Thanks for all you guys do any advice is much appreciated

**Comment by TheMindExpansionNetwork** [comment]
https://huggingface.co/bigscience/bloom

**Reply by BarfingLemurs** [reply]
I loaded bloom-z before, with no problems. you can load many hf models as long as you pick the appropriate sizes.
